LayoutTests/imported/w3c/web-platform-tests/css/css-syntax/custom-property-rule-ambiguity-expected.txt

```diff
@@ -1,6 +1,6 @@
 
-FAIL Rule that looks like a custom property declaration is ignored assert_equals: expected 2 but got 3
-FAIL Rule that looks like an invalid custom property declaration is ignored assert_equals: expected 2 but got 3
+PASS Rule that looks like a custom property declaration is ignored
+PASS Rule that looks like an invalid custom property declaration is ignored
 FAIL Nested rule that looks like a custom property declaration assert_equals: expected "hover { }\n    .b { }" but got "hover { }     .b { }"
```

Comment:
Are you planning to address this?

Reply:
I've opened a bug https://bugs.webkit.org/show_bug.cgi?id=276431